### PR TITLE
Feature fix

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Fixed issues with styling and classes

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -73,9 +76,93 @@ main {
 footer {
     text-align: center;
     grid-area: footer;
-    color: var(--color);
+    color: var(--color) ;
     background-color: var(--back-color);
     border-top-left-radius: 0.5rem;
     border-top-right-radius: 0.5rem;
 }
+/*
```
Comment (shaksham08) [inline style.css]: is this commented code needed?

**Review by shaksham08 — APPROVED** [review]
Looks good , just added one comment

**Review by sushil-mod — APPROVED** [review]
fixes look good , you can merge

File: style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply style.css]: No, I forgot to delete them. I will delete

**Pull request merged**